org.inkscape.Inkscape.json

```diff
@@ -0,0 +1,192 @@
+{
+    "app-id": "org.inkscape.Inkscape",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.20",
+    "sdk": "org.gnome.Sdk",
+    "command": "inkscape",
+    "rename-desktop-file": "inkscape.desktop",
+    "rename-appdata": "inkscape",
+    "rename-icon": "inkscape",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--filesystem=host"
+    ],
+    "build-options": {
+        "cflags": "-O2",
+        "cxxflags": "-O2 -std=c++14",
+        "env": {
+            "V": "1"
+        }
+    },
+    "cleanup": [
+        "/bin/gsl-*",
+        "/include",
+        "/lib/pkgconfig",
+        "/share/aclocal",
+        "/share/devhelp",
+        "/share/doc",
+        "/share/gc",
+        "/share/info",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "boost",
+            "config-opts": [],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.sourceforge.net/project/boost/boost/1.60.0/boost_1_60_0.tar.bz2",
+                    "sha256": "686affff989ac2488f79a97b9479efb9f2abae035b5ed4d8226de6857933fd3b"
+                },
+                {
```

Comment:
There is a `simple` build system now, that takes commands from `build-commands` array; is there any benefit from faking autotools?